frontend/src/views/FuelSupplies/_schema.jsx

```diff
@@ -416,12 +419,13 @@ export const fuelSupplyColDefs = (optionsData, errors, warnings) => [
       params.value ||
       (!params.value && <Typography variant="body4">Select</Typography>),
     suppressKeyboardEvent,
-    editable: (params) => /other/i.test(params.data.fuelType),
+    editable: (params) => params.data.fuelType === FUEL_TYPE_OTHER,
```

Comment:
Hard-coding string constants across multiple frontend places and the backend creates fragile code. My idea would be to add a Boolean to fuel_type table that controls this field instead. Thoughts?

Reply:
I’ve refactored the code so the hard-coded string is only checked in one place within the file. I’m not sure adding a boolean field to the fuel_type table is warranted since it would only apply to the 'Other' record.